src/idiom259ex.fz

```diff
@@ -0,0 +1,5 @@
+ex259 is
+  strings := ["a-b_c,d", "-a--cc_,eee,ffff_", "---", ",_-", "abcd", ""]
+
+  _ := for s in strings do
+          say "\"$s\" : {s.replace("_", ",").replace("-", ",").split(",")}"
```

Comment:
better use exactly the code from `idiom259.fuzion`, i.e., `parts := s.replace...`